features/dd-sdk-android-trace/src/main/java/com/datadog/opentelemetry/trace/OtelSpanBuilder.java

```diff
@@ -21,168 +21,176 @@
 import com.datadog.android.api.InternalLogger;
 import com.datadog.trace.bootstrap.instrumentation.api.AgentSpan;
 import com.datadog.trace.bootstrap.instrumentation.api.AgentTracer;
+
 import io.opentelemetry.api.common.AttributeKey;
 import io.opentelemetry.api.common.Attributes;
 import io.opentelemetry.api.trace.Span;
 import io.opentelemetry.api.trace.SpanBuilder;
 import io.opentelemetry.api.trace.SpanContext;
 import io.opentelemetry.api.trace.SpanKind;
 import io.opentelemetry.context.Context;
+
 import java.util.List;
 import java.util.concurrent.TimeUnit;
 import java.util.logging.Logger;
 
 public class OtelSpanBuilder implements SpanBuilder {
```

Comment:
these diffs are mostly from new format applied because this was copied from the Java agent project.